src/test/java/org/jabref/logic/pdf/PdfAnnotationImporterTest.java

```diff
@@ -49,38 +49,34 @@ public void noAnnotationsMinimal() {
 
     @Test
     public void inlineNoteMinimal() {
-        List<FileAnnotation> annotations = importer
-                .importAnnotations(Paths.get("src/test/resources/pdfs/minimal-inlinenote.pdf"));
-
-        // Begin old test
-        assertEquals(1, annotations.size());
-        FileAnnotation note = annotations.get(0);
-        assertEquals("Linus Dietz", note.getAuthor());
-        assertEquals(LocalDateTime.of(2017, 3, 12, 20, 25), note.getTimeModified());
-        assertEquals(1, note.getPage());
-        assertEquals("inline note annotation", note.getContent());
-        assertEquals("FreeText", note.getAnnotationType());
-        assertFalse(note.hasLinkedAnnotation());
-        // End old test
-
-        FileAnnotation expected = new FileAnnotation("Linus Dietz", LocalDateTime.of(2017, 3, 12, 20, 25), 1,
-                "inline note annotation", "FreeText", Optional.empty());
-        assertEquals(Collections.singletonList(expected), annotations);
+        final List<FileAnnotation> expected = Collections.singletonList(
+                new FileAnnotation("Linus Dietz", LocalDateTime.of(2017, 3, 12, 20, 25), 1,
+                        "inline note annotation", "FreeText", Optional.empty()));
 
+        assertEquals(expected,
+                importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal-inlinenote.pdf")));
     }
 
     @Test
     public void popupNoteMinimal() {
-        List<FileAnnotation> annotations = importer
-                .importAnnotations(Paths.get("src/test/resources/pdfs/minimal-popup.pdf"));
-        assertEquals(1, annotations.size());
+        // Fixme: Comparison Fails because seconds are missing, should be 24.
+        final List<FileAnnotation> expected = Collections.singletonList(
+                new FileAnnotation("Linus Dietz", LocalDateTime.of(2017, 3, 12, 20, 17), 1,
+                        "A simple pop-up note", "Text", Optional.empty()));
 
+        // Start old test style
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal-popup.pdf"));
         FileAnnotation note = annotations.get(0);
         assertEquals("Linus Dietz", note.getAuthor());
+        assertEquals(LocalDateTime.of(2017, 3, 12, 20, 17, 24), note.getTimeModified());
+        assertEquals(1, note.getPage());
+
         assertEquals("A simple pop-up note", note.getContent());
-        assertFalse(note.hasLinkedAnnotation());
         assertEquals("Text", note.getAnnotationType());
-        assertEquals(1, note.getPage());
+        // End old test style
+
+        assertEquals(expected,
```

Comment:
To see the value of comparison with lists, we need to assume that the method under test returns something unexpected. For example:
- It returns two items instead of one. With the "old style" you just get `expected 1 but got 2` while comparison with the singleton list displays something like `Expected "item [property 1, ...]" but got "item [property 1, ....], item2 [value 1, ... ]"`. So you get more details about the item that is wrongly returned, which is often helpful in fixing bugs. 
- The method still returns only one item but with the wrong values. In this case, the old style gives you `expected "value1" but got "wrong value"`. If you compare against an expected item you directly get also information about the other properties (thus you directly see if just one value is off or a completely different item is returned). 

Another important advantage is that, if in the future the class is extended by an additional property, you just need to adapt the equals method and all your tests automatically take this new property in account.